api/graphql/schema.graphql

```diff
@@ -143,35 +143,36 @@ type Department {
     name: LocalizedString
 }
 
-enum Status {
+enum PoolCandidateSearchStatus {
     PENDING
     DONE
 }
 
 type PoolCandidateFilter {
     id: ID!
     classifications: [Classification]
-    cmoAssets: [CmoAssets]
+    cmoAssets: [CmoAsset]
     hasDiploma: Boolean @rename(attribute: "has_diploma")
     hasDisability: Boolean @rename(attribute: "has_disability")
     isIndigenous: Boolean @rename(attribute: "is_indigenous")
     isVisibleMinority: Boolean @rename(attribute: "is_visible_minority")
     isWoman: Boolean @rename(attribute: "is_woman")
     languageAbility: LanguageAbility @rename(attribute: "language_ability")
     operationalRequirements: [OperationalRequirement] @rename(attribute: "operational_requirements")
-    workRegion: [WorkRegion] @rename(attribute: "work_region")
+    workRegions: [WorkRegion] @rename(attribute: "work_regions")
+    pools: [Pool]
 }
 
 type PoolCandidateSearchRequest {
     id: ID!
-    fullName: String! @rename(attribute: "full_name")
-    email: String!
-    department: Department! @hasOne
-    jobTitle: String! @rename(attribute: "job_title")
+    fullName: String @rename(attribute: "full_name")
+    email: String
+    department: Department @hasOne
+    jobTitle: String @rename(attribute: "job_title")
     additionalComments: String @rename(attribute: "additional_comments")
-    poolCandidateFilters: PoolCandidateFilter! @hasOne
-    requestedDate: Date! @rename(attribute: "requested_date")
-    status: Status!
+    poolCandidateFilterId: PoolCandidateFilter! @hasOne @rename(attribute: "pool_candidate_filter_id")
```

Comment:
After running `php artisan lighthouse:validate-schema,` this error is returned:

>Node poolCandidateFilterId can only have one directive of type Nuwave\Lighthouse\Support\Contracts\FieldResolver but found [`@hasOne`, `@rename`].

I looked at the docs for the [hasOne directive](https://lighthouse-php.com/5/api-reference/directives.html#hasone) and the following might work (thought not exactly sure since the type `PoolCandidateSearchRequest` isn't used anywhere yet):
`@hasOne @rename(attribute: "pool_candidate_filter_id")`
to
`@hasOne(relation: "pool_candidate_filter_id")`

Thoughts @tristan-orourke?

Reply:
I removed the rename directive as it's not needed in this case, and correctly changed the name to poolCandidateFilter (from poolCandidateFilterId)

Reply:
Schema is valid on my end now.